CE09OSPM/CE09OSPM_D00018_ingest.csv

```diff
@@ -0,0 +1,8 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+# mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CE09OSPM/D00018/irid2shore/stc_status.*_*.txt,CE09OSPM-SBS01-00-STCENG000,telemetered,Pending,Incorrectly specified dataset and parser
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CE09OSPM/D00018/imm/mmp/E*.DAT,CE09OSPM-WFP01-00-WFPENG000,telemetered,Available,
```

Comment:
on the raw data server, using D00017 as an example data directory structure for OSPM, it looks like there should be a "cg_data" directory between "D00018" and the "imm" directory, unless I'm missing something.